includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
```

Comment:
@nimid so, I still prefer to implement with the text domain as below
```php
$text_domain = 'Omise';

__( 'Enable/Disable', $text_domain );
```

Because we can't guarantee that no one (developers) who's downloaded our plugin will not implement `I18n` feature with the text domain `Omise` by themselves, so if we remove `__()` method it may break some website that they already implemented `I18n`.